subsys/bluetooth/audio/Kconfig

```diff
@@ -17,31 +17,42 @@ menuconfig BT_AUDIO
 
 if BT_AUDIO
 
-if BT_CONN
-
 config BT_AUDIO_UNICAST
-	bool "Bluetooth Unicast Audio Support"
+	bool # Hidden
```

Comment:
Personally I'm a fan of using `# Hidden` for Kconfigs, but I think that someone just pruned a lot of these upstream as it apparently is implied.

Reply:
Do I need to change it?

Reply:
Not for my sake; it's up to you, but we may get requested to remove it when we move this in the `main` branch.